### DISCUSSION TITLE
What do I need to backup in order to be able to recover in case my node dies and the server catches fire 🔥?

### DISCUSSION THREAD

**Discussion by FeatureSpitter** [lightningnetwork/lnd#7948] [Q&A]
Backing up a LN node seems complex.

And I was wondering... Besides copying the .lnd folder and its contents, if I activate Postgres as a database, would backing up the database **alone** be enough? Or I still need the .lnd folder?

Thanks

**Comment by guggero** [comment]
See [the lnd Operational Safety Guidelines document](https://github.com/lightningnetwork/lnd/blob/master/docs/safety.md).
It was written before Postgres was a thing, but in general, the Postgres DB contains the data that otherwise would be in `wallet.db`, `macaroons.db` and `channel.db`.

Backing up the DB is always a good idea. But restoring it from a backup that is not the absolute very last state (which is almost impossible to know) can be dangerous (see explanation in linked doc above).
